tests/test_sqlite.py

```diff
@@ -25,3 +27,4 @@
 }
 
 INSTALLED_APPS = ["django.contrib.sessions"]
+DJANGO_REDIS_CLOSE_CONNECTION = False
```

Comment:
I think this should not be added, even if it does not do any harm, otherwise all the default values has to be added in all the tests, which I think it is not worth it

Reply:
I fix.